### ISSUE TITLE
!skip command is not working

### ISSUE THREAD

**Issue by PRiMENON** [Just-Some-Bots/MusicBot#1918]
sorry, I cannot speak good english. I use google translate.

Please tick all applicable boxes.

- [x] I am using Python 3.5.3 or higher (run `python --version` on the command line)
- [x] I have followed the [official guides](https://github.com/Just-Some-Bots/MusicBot/wiki) to install the bot for my system
- [x] I have updated my dependencies to the latest version using the appropriate `update` script

Which version are you using?

- [x] The latest `master` version (release-120519)
- [ ] The latest `review` version

What type of issue are you creating?

- [ ] Bug
- [ ] Feature request
- [x] Question

----

### Description of issue
`!skip` command is not working in my computer.

### Steps to reproduce
 1. click to "update.bat"
 1. join to my voice chat on discord.
 1. click to "run.bat"
 1. on discord, type to "!summon"
 1. playing music... and type to "!skip" or "!skip f"

### Actual Results
stop music, but not play next music.

run.bat respond following message:
```
[ERROR:player] Can't delete file, it is currently in use: {0}
```

### Expected Results
stop music and next playing music.

### system info
 * MusicBot release-120519
 * Windows 10 Home version 1809(64bit)
 * python (64bit) v3.7.3
 * Git for Windows v2.21.0
 * TortoiseGit v2.8.0.0(64bit)

----

### Log file
<details>
<summary>logfiles</summary>

```
[46.891450882] 2019-06-09 17:30:30,881 - INFO - launcher: Starting sanity checks
[46.891450882] 2019-06-09 17:30:30,881 - INFO - launcher: Checking for Python 3.5+
[46.891450882] 2019-06-09 17:30:30,881 - INFO - launcher: Checking console encoding
[46.891450882] 2019-06-09 17:30:30,881 - INFO - launcher: Setting console encoding to UTF-8
[46.891450882] 2019-06-09 17:30:30,881 - INFO - launcher: Ensuring we're in the right environment
[46.891450882] 2019-06-09 17:30:30,881 - INFO - launcher: Adding local bins/ folder to path
[685.044527054] 2019-06-09 17:30:31,520 - INFO - launcher: Required checks passed.
[685.044527054] 2019-06-09 17:30:31,520 - INFO - launcher: Optional checks passed.

######################### PRE-RUN SANITY CHECKS PASSED #########################

[1.4662511348724365] 2019-06-09 17:30:32,301 - WARNING - musicbot.config | In config.py::MainThread(5164), line 120 in run_checks: i18n file does not exist. Trying to fallback to config/i18n/en.json.
[1.4662511348724365] 2019-06-09 17:30:32,301 - INFO - musicbot.config | In config.py::MainThread(5164), line 131 in run_checks: Using i18n: config/i18n/en.json
[1.4662511348724365] 2019-06-09 17:30:32,301 - DEBUG - musicbot.bot | In bot.py::MainThread(5164), line 226 in _setup_logging: Set logging level to INFO
[1.4662511348724365] 2019-06-09 17:30:32,301 - DEBUG - musicbot.json | In json.py::MainThread(5164), line 8 in __init__: Init JSON obj with config/i18n/en.json
[1.4974780082702637] 2019-06-09 17:30:32,332 - INFO - musicbot.bot | In bot.py::MainThread(5164), line 91 in __init__: Starting MusicBot release-120519
[1.4974780082702637] 2019-06-09 17:30:32,332 - WARNING - musicbot.bot | In bot.py::MainThread(5164), line 94 in __init__: Autoplaylist is empty, disabling.
[4.5306255817413330] 2019-06-09 17:30:35,365 - DEBUG - musicbot.bot | In bot.py::MainThread(5164), line 949 in on_ready: Connection established, ready to go.
[4.5306255817413330] 2019-06-09 17:30:35,365 - DEBUG - musicbot.bot | In bot.py::MainThread(5164), line 340 in _cache_app_info: Caching app info
[4.7493400573730469] 2019-06-09 17:30:35,584 - DEBUG - musicbot.bot | In bot.py::MainThread(5164), line 771 in _scheck_ensure_env: Ensuring data folders exist
[4.7804615497589111] 2019-06-09 17:30:35,615 - DEBUG - musicbot.bot | In bot.py::MainThread(5164), line 781 in _scheck_ensure_env: Deleted old audio cache
[4.7804615497589111] 2019-06-09 17:30:35,615 - DEBUG - musicbot.bot | In bot.py::MainThread(5164), line 787 in _scheck_server_permissions: Checking server permissions
[4.7804615497589111] 2019-06-09 17:30:35,615 - DEBUG - musicbot.bot | In bot.py::MainThread(5164), line 791 in _scheck_autoplaylist: Auditing autoplaylist
[4.7804615497589111] 2019-06-09 17:30:35,615 - DEBUG - musicbot.bot | In bot.py::MainThread(5164), line 795 in _scheck_configs: Validating config
[4.7804615497589111] 2019-06-09 17:30:35,615 - DEBUG - musicbot.config | In config.py::MainThread(5164), line 228 in async_validate: Validating options...
[4.7804615497589111] 2019-06-09 17:30:35,615 - DEBUG - musicbot.config | In config.py::MainThread(5164), line 242 in async_validate: Acquired owner id via API
[4.7804615497589111] 2019-06-09 17:30:35,615 - DEBUG - musicbot.bot | In bot.py::MainThread(5164), line 798 in _scheck_configs: Validating permissions config
[4.7804615497589111] 2019-06-09 17:30:35,615 - DEBUG - musicbot.permissions | In permissions.py::MainThread(5164), line 94 in async_validate: Validating permissions...
[4.7804615497589111] 2019-06-09 17:30:35,615 - DEBUG - musicbot.permissions | In permissions.py::MainThread(5164), line 98 in async_validate: Fixing automatic owner group
[4.7804615497589111] 2019-06-09 17:30:35,615 - INFO - musicbot.bot | In bot.py::MainThread(5164), line 966 in on_ready: Connected: <<MASKED>>
[4.7804615497589111] 2019-06-09 17:30:35,615 - INFO - musicbot.bot | In bot.py::MainThread(5164), line 974 in on_ready: Owner:     <<MASKED>>

[4.7960753440856934] 2019-06-09 17:30:35,631 - INFO - musicbot.bot | In bot.py::MainThread(5164), line 977 in on_ready: Guild List:
[4.7960753440856934] 2019-06-09 17:30:35,631 - INFO - musicbot.bot | In bot.py::MainThread(5164), line 981 in on_ready:  - <<MASKED>>
[4.7960753440856934] 2019-06-09 17:30:35,631 - INFO - musicbot.bot | In bot.py::MainThread(5164), line 1037 in on_ready: Not bound to any text channels
[4.7960753440856934] 2019-06-09 17:30:35,631 - INFO - musicbot.bot | In bot.py::MainThread(5164), line 1063 in on_ready: Not autojoining any voice channels
[17.9872119426727295] 2019-06-09 17:30:48,822 - INFO - musicbot.bot | In bot.py::MainThread(5164), line 2680 in on_message: <<MASKED>>: !summon
[18.9030530452728271] 2019-06-09 17:30:49,738 - DEBUG - musicbot.bot | In bot.py::MainThread(5164), line 731 in deserialize_queue: Deserializing queue for <<MASKED>>
[19.3109490871429443] 2019-06-09 17:30:50,146 - DEBUG - musicbot.bot | In bot.py::MainThread(5164), line 423 in get_player: Created player via deserialization for guild <<MASKED>> with 11 entries
[19.3109490871429443] 2019-06-09 17:30:50,146 - INFO - musicbot.bot | In bot.py::MainThread(5164), line 1926 in cmd_summon: Joining <<MASKED>>
[19.3149478435516357] 2019-06-09 17:30:50,150 - DEBUG - musicbot.entry | In entry.py::MainThread(5164), line 56 in get_ready_future: Created future for None
[19.3149478435516357] 2019-06-09 17:30:50,150 - DEBUG - musicbot.entry | In entry.py::MainThread(5164), line 56 in get_ready_future: Created future for None
[19.3149478435516357] 2019-06-09 17:30:50,150 - INFO - musicbot.entry | In entry.py::MainThread(5164), line 292 in _really_download: Download started: https://www.youtube.com/watch?v=<<MASKED>>
[19.3589358329772949] 2019-06-09 17:30:50,194 - INFO - musicbot.entry | In entry.py::MainThread(5164), line 292 in _really_download: Download started: https://www.youtube.com/watch?v=<<MASKED>>
[23.1516776084899902] 2019-06-09 17:30:53,986 - INFO - musicbot.entry | In entry.py::MainThread(5164), line 302 in _really_download: Download complete: https://www.youtube.com/watch?v=<<MASKED>>
[31.5106363296508789] 2019-06-09 17:31:02,345 - INFO - musicbot.entry | In entry.py::MainThread(5164), line 302 in _really_download: Download complete: https://www.youtube.com/watch?v=<<MASKED>>
[31.5266189575195312] 2019-06-09 17:31:02,361 - FFMPEG - musicbot.player | In player.py::MainThread(5164), line 290 in _play: Creating player with options: -nostdin -vn audio_cache\<<MASKED>>.webm
[31.5634555816650391] 2019-06-09 17:31:02,398 - DEBUG - musicbot.player | In player.py::MainThread(5164), line 303 in _play: Playing <musicbot.player.SourcePlaybackCounter object at 0x000001F557C8DAC8> using <discord.voice_client.VoiceClient object at 0x000001F557999E80>
[31.5674695968627930] 2019-06-09 17:31:02,402 - DEBUG - musicbot.bot | In bot.py::MainThread(5164), line 458 in on_player_play: Running on_player_play
[31.5674695968627930] 2019-06-09 17:31:02,402 - DEBUG - musicbot.bot | In bot.py::MainThread(5164), line 707 in serialize_queue: Serializing queue for <<MASKED>>
[35.3307523727416992] 2019-06-09 17:31:06,165 - INFO - musicbot.bot | In bot.py::MainThread(5164), line 2680 in on_message: <<MASKED>>: !skip
[35.3473179340362549] 2019-06-09 17:31:06,182 - DEBUG - musicbot.player | In player.py::Thread-3(8296), line 217 in _playback_finished: Deleting file: audio_cache\<<MASKED>>.webm
[35.3512830734252930] 2019-06-09 17:31:06,186 - ERROR - musicbot.player | In player.py::Thread-3(8296), line 226 in _playback_finished: Can't delete file, it is currently in use: {0}
[43.4704365730285645] 2019-06-09 17:31:14,305 - INFO - musicbot.bot | In bot.py::MainThread(5164), line 2680 in on_message: <<MASKED>>: !shutdown
[43.990507603] launcher-INFO: All done.
```
</details>

### data\options.ini only edited
<details>
<summary>data\options.ini only edited</summary>

```
[Credentials]
Token = <<MASKSED>

(snip)

[MusicBot]
SaveVideos = no
AutoSummon = no
UseAutoPlaylist = no
```
</details>

### data\permissons.ini only edited
<details>
<summary>data\permissons.ini only edited</summary>

```
[MusicMaster]
GrantToRoles = <<MASKED>>
```
</details>

### message from run.bat
<details>
<summary>message from run.bat</summary>

```
[INFO] launcher: Starting sanity checks
[INFO] launcher: Checking for Python 3.5+
[INFO] launcher: Checking console encoding
[INFO] launcher: Setting console encoding to UTF-8
[INFO] launcher: Ensuring we're in the right environment
[INFO] launcher: Adding local bins/ folder to path
[INFO] launcher: Required checks passed.
[INFO] launcher: Optional checks passed.
[INFO] launcher: Moving old musicbot log

Starting MusicBot release-120519
WARNING: Autoplaylist is empty, disabling.

Connected: <<MASKED>>
Owner:     <<MASKED>>

Guild List:
 - <<MASKED>>

Not bound to any text channels
Not autojoining any voice channels

<<MASKED>>: !summon
Joining <<MASKED>>
Download started: https://www.youtube.com/watch?v=<<MASKED>>
Download started: https://www.youtube.com/watch?v=<<MASKED>>
Download complete: https://www.youtube.com/watch?v=<<MASKED>>
Download complete: https://www.youtube.com/watch?v=<<MASKED>>
<<MASKED>>: !skip f
[ERROR:player] Can't delete file, it is currently in use: {0}
```
</details>

thanks.

**Comment by TheerapakG** [comment]
Known issue, fix proposed #1912

**Comment by PRiMENON** [comment]
@TheerapakG thank you! 😄

**Comment by Doorstop2** [comment]
yeah having the same problem

**Comment by ghost** [comment]
Fixed with #1912, checkout the review branch. :)
